src/Pages/Profile/admin/AdminView.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+//import "./profile.css";
+import {
+  Button,
+  Modal,
+} from "reactstrap";
+import axios from 'axios'
+import Display from './Profile.js'
+import EditForm from './EditorForm'
+const bcrypt = require('bcrypt-nodejs')
+
+export default function Editor(props) {
+  // first name, last name, middle initial, email, pass, door code
+  const [firstName, setFirstName] = useState("")
+  const [lastName, setLastName] = useState("")
+  const [middleInitial, setMiddleInitial] = useState("")
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [doorCode, setDoorCode] = useState("")
+  const [major, setMajor] = useState("")
+  const [user, setUser] = useState({ ...props.user })
+  const [toggle, setToggle] = useState(false)
+  const [pagesPrinted, setPagesPrinted] = useState(user.pagesPrinted)
+  const [toggleSubmit, setToggleSubmit] = useState(false)
+  const [userMembership, setuserMembership] = useState(user.accessLevel)
+  const [numberOfSemestersToSignUpFor, setNumberOfSemestersToSignUpFor] = useState(undefined)
+  const [membershipValidUntil, setMembershipValidUntil] = useState(user.membershipValidUntil)
+
+  function handleToggle() {
+    setToggle(!toggle)
+  }
+
+  async function handleSubmission() {
+    const queryEmail = user.email
+
+    //hash pass
+    const salt = bcrypt.genSaltSync(10)
+    const hashed = (password.trim() === '') ?
+      user.password : bcrypt.hashSync(password, salt)
+
+    //Change term?
+    numberOfSemestersToSignUpFor === 0 &&
```

Comment:
what's the point of this line?

Reply:
the new updates just make this useless i will delete them